packages/edge-login-ui-rn/src/common/util/PixelUtil.js

```diff
@@ -1,16 +1,18 @@
+// @flow
 import { Dimensions, PixelRatio } from 'react-native'
 const { width, height } = Dimensions.get('window')
 
 // Guideline sizes are based on standard ~5" screen mobile device
 const guidelineBaseWidth = 320
 const guidelineBaseHeight = 480
 
-const hs = size => width / guidelineBaseWidth * size // horizontal scale
-const vs = size => height / guidelineBaseHeight * size // verical scale
-const ms = (size, factor = 0.5) => size + (hs(size) - size) * factor // moderate scale.... this allows you to adjust the refactor size
+const hs = (size: number) => width / guidelineBaseWidth * size // horizontal scale
```

Comment:
@ahujsak This utility has me intrigued. You will have to share what you are doing with me. Is this only for text or ?

Reply:
So, it is for font size, based on the the pixel density of the phone. I have an outstanding task to get the font sizes right, and the font part of this will be used. Right now, nothing is using the utility in the code.